encoding/video-artifacts.md

```diff
@@ -232,6 +234,29 @@ or a manual filtering of dehalo\_alpha with dhhmask
 
 [![field-noise.jpg](images/3cnvimage103.png)](https://diff.pics/84URvW5IYSdO/1)
 
+
+### Underflow/Overflow
+
+While most of the anime produced
+use the YUV 8-bit limited range[^4],
+we occasionally find some videos having the "limited range" flag set
+while containing full range content.
+This often results in oversaturated colors and weird brightness.
+Thus, it is strongly recommended
+to check the brightness levels with `hist.Levels()`.
```

Comment:
This should probably be applied to the 8-bit source clip, correct?

Reply:
Yes! hist.Levels() only works on 8bit, we should mention that